docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -164,7 +164,15 @@ Dates listed indicate the start date of the maintenance cycle.
      - End Of Life
      - Control Node Python
      - Target Python / PowerShell
-   * - `2.17`_
+   * - `2.18`_
+     - | GA: 04 Nov 2024
+       | Critical: 19 May 2025
+       | Security: 03 Nov 2025
+     - May 2026
+     - | Python 3.10 - 3.13
+     - | Python 3.8 - 3.13
+       | PowerShell 5.1
+    * - `2.17`_
```

Comment:
```suggestion
   * - `2.18`_
     - | GA: 04 Nov 2024
       | Critical: 19 May 2025
       | Security: 03 Nov 2025
     - May 2026
     - | Python 3.10 - 3.13
     - | Python 3.8 - 3.13
       | PowerShell 5.1
   * - `2.17`_
```

Reply:
Not sure if it has any effect or not but it seems the indentation on that one line is off by a space. Could just be me though.

Reply:
Ah, I think this does have an impact. Looks like ci is failing with the RST check and docs build.

Reply:
Apologies, just pushed a change.  In my editor, everything in the table all shows up in the same color, and couldn't spot the indentation issue immediately.